generators/app/templates/package.json

```diff
@@ -26,7 +26,8 @@
   },
   "peerDependencies": {
     "react": "^15.4.2",
-    "react-dom": "^15.4.2"
+    "react-dom": "^15.4.2",
+    "terra-base": "^0.x"
```

Comment:
This is pretty confusing and I don't know if there is a good rule. If anything is listed as a dependency, there is some chance that the package could be duplicated in a webpack bundle.

I think the best we can do is follow the same pattern as other libraries. We should keep an eye on our bundles for duplication and fix it where possible. Applications will still need to be aware of this and use [resolve.alias](https://webpack.js.org/configuration/resolve/#resolve-alias) where necessary